farao-dichotomy-api/src/test/java/com/farao_community/farao/dichotomy/api/NetworkValidatorMock.java

```diff
@@ -21,7 +21,12 @@ public NetworkValidatorMock(double limit) {
     }
 
     @Override
-    public DichotomyStepResult<Object> validateNetwork(Network network) throws ValidationException {
+    public DichotomyStepResult<Object> validateNetwork(Network network) {
```

Comment:
i would remove or deprecate it. I think breaking compatibility, still better than breaking behaviour..

Reply:
obsolete